synapse/storage/databases/main/events.py

```diff
@@ -1641,7 +1641,7 @@ def _store_room_members_txn(self, txn, events, backfilled):
         """Store a room member in the database."""
 
         def str_or_none(val: Any) -> Optional[str]:
```

Comment:
I wonder if we should update the name of this now that it does more than ensuring it is a string? 🤷 It's a pretty simple function so not a big deal.

Reply:
I updated the name, let me know if you think it's a little clearer.